### PR TITLE
Support disk limits in projects

### PR DESCRIPTION
**Pull request by freeekanayaka**

### CODE REVIEW

**Comment by stgraber** [comment]
Oh wow, how did I miss that leftover debugging ;)

File: doc/projects.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ features.storage.volumes             | boolean   | -                     | true
 limits.containers                    | integer   | -                     | -                         | Maximum number of containers that can be created in the project
 limits.virtual-machines              | integer   | -                     | -                         | Maximum number of VMs that can be created in the project
 limits.cpu                           | integer   | -                     | -                         | Maximum value for the sum of individual "limits.cpu" configs set on the instances of the project
-limits.memory                        | integer   | -                     | -                         | Maximum value for the sum of individual "limits.memory" configs set on the instances of the project
+limits.disk                          | string    | -                     | -                         | Maximum value for the sum of individual "limits.memory" configs set on the instances of the project
```
Comment (stgraber) [inline doc/projects.md:26]: Description need update

Comment (stgraber) [reply doc/projects.md:26]: More like, got swapped with limits.memory :)

**Comment by stgraber** [comment]
@freeekanayaka looks good overall. Adding custom storage volumes should be pretty straightforward, adding images will need a bit more work due to the way they're getting written (unknown size so need a WriteCloser with a size budget).

File: doc/projects.md (hunk 1 above)
Comment (freeekanayaka) [reply doc/projects.md]: Ah yeah :) Fixed.

**Comment by stgraber** [comment]
Pushed the drive-by fix directly to master.

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
jenkins: test this please

**Pull request merged**